Sources/Controllers/TargetMenu/Search/OAQuickSearchCoordinateFormatsViewController.m

```diff
@@ -106,40 +103,36 @@ - (void) viewDidLoad
 {
     [super viewDidLoad];
     
+    self.navigationItem.title = OALocalizedString(@"coords_format");
     self.tableView.delegate = self;
     self.tableView.dataSource = self;
     [self.tableView registerClass:OATableViewCustomFooterView.class forHeaderFooterViewReuseIdentifier:[OATableViewCustomFooterView getCellIdentifier]];
     self.tableView.separatorColor = UIColor.separatorColor;
-    self.tableView.contentInset = UIEdgeInsetsMake(defaultNavBarHeight, 0, 0, 0);
     self.tableView.tintColor = UIColor.iconColorActive;
-    self.doneButton.hidden = YES;
-    self.doneButton.enabled = NO;
-
-    self.titleLabel.text = OALocalizedString(@"coords_format");
-    [self.cancelButton setTitle:OALocalizedString(@"shared_string_back") forState:UIControlStateNormal];
-    
-    _navBarBackgroundView = [self createNavBarBackgroundView];
-    _navBarBackgroundView.frame = self.navbarView.bounds;
-    [self.navbarView insertSubview:_navBarBackgroundView atIndex:0];
 }
 
-- (UIView *) createNavBarBackgroundView
+- (void) viewWillAppear:(BOOL)animated
 {
-    if (!UIAccessibilityIsReduceTransparencyEnabled())
-    {
-        UIVisualEffectView *blurEffectView = [[UIVisualEffectView alloc] initWithEffect:[UIBlurEffect effectWithStyle:[ThemeManager shared].isLightTheme ? UIBlurEffectStyleSystemUltraThinMaterialLight : UIBlurEffectStyleSystemUltraThinMaterialDark]];
-        blurEffectView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-        blurEffectView.alpha = 0;
-        return blurEffectView;
-    }
-    else
-    {
-        UIView *res = [[UIView alloc] init];
-        res.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
-        res.backgroundColor = UIColor.viewBgColor;
-        res.alpha = 0;
-        return res;
-    }
+    [super viewWillAppear:animated];
+    
+    [self.navigationController setNavigationBarHidden:NO animated:YES];
```

Comment:
The code for styling the navigation bar should be placed in a separate method, configureNavigationBar